src/codeu/chat/server/Controller.java

```diff
@@ -140,6 +177,11 @@ public ConversationHeader newConversation(Uuid id, String title, Uuid owner, Tim
       LOG.info("Conversation added: " + id);
     }
 
+    //Log should load before adding new commands.
```

Comment:
I'd say maybe put the two segments under the one "//Log should load before adding new commands." comment